temporalnexus/operation.go

```diff
@@ -262,11 +252,14 @@ type WorkflowHandle[T any] interface {
 	ID() string
 	// ID is the workflow's run ID.
 	RunID() string
+	// Link to the WorkflowExecutionStarted event of the workflow represented by this handle.
+	Link() nexus.Link
```

Comment:
hm why does this need to be on the interface? Are users expected to use this method?

Reply:
It doesn't but I don't think it hurts.

Another approach would be to put this on the struct implementation and have an unexported method on the interface to ensure users don't try to implement it. This is important since the handler's return value is the interface.

Reply:
I'll change to that alternative.

Reply:
Done.